### PR TITLE
Create negative Tests

### PR DESCRIPTION
**Pull request by Oleg-Decik**

### CODE REVIEW

**Review by SET-SKYNET — APPROVED** [review]

File: allure-results/6fcb3992-a2ac-4201-9821-8cf29c9156b0-container.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"uuid":"6fcb3992-a2ac-4201-9821-8cf29c9156b0","name":"Verify that immediately after new user creation, User's name is shown in the header","children":["8284818e-a0eb-47af-97e5-e0df66b95984"],"befores":[{"name":"setUpWebDriver","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1573824786258,"stop":1573824798162}],"afters":[{"name":"closeWebDriver","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1573824810195,"stop":1573824810883}],"links":[],"start":1573824786227,"stop":1573824810883}
```
Comment (ostret) [inline allure-results/6fcb3992-a2ac-4201-9821-8cf29c9156b0-container.json:1]: what's that file doing here ?

File: src/main/java/helperinstrument/LogInHelper.java [hunk 2]
```diff
@@ -25,7 +24,14 @@ public void initLogUserInfo() {
...
-
+    }
+    public void initWrongLogUserInfo() {
+        
+        userPage = new LogUserInformation()
+                .getUnloggedUserInfo()
+                .clickSignInHeaderFull()
+                .getLoginForm()
+                .logIn( "some@gmail.com", "somes" );
     }
```
Comment (ostret) [inline src/main/java/helperinstrument/LogInHelper.java:35]: this is almost exactly the same as method above
please refactor

File: src/main/java/helperInstrument/EnviromentHelper.java [hunk 3]
```diff
@@ -31,6 +31,11 @@ public static String getMasterPass () {
 		return masterPass;
 	}
 	
+	public static String getAlternativePass(String alternativePass){
+		masterPass = alternativePass;
```
Comment (ostret) [inline src/main/java/helperInstrument/EnviromentHelper.java:35]: why are you setting something in get method ???

File: README.md [hunk 4]
```diff
@@ -22,4 +22,8 @@ Report will be generated based on /target/allure-results dir and placed to targe
 Open index.html in any browser <br>
 
 **Environment variables**
+Using environment variables:
+1. Tap edit Run/Debug configurations menu
+2. Click 'edit configurations'
+3. On VM options bar tap all needed environments variable
 // TBD
```
Comment (ostret) [inline README.md:29]: list a name of a envVariable used, e.g. prestoUrl and give a hint of what it's supposed to be

Comment (ostret) [reply README.md:29]: fix mvn command above

**Review by SET-SKYNET — APPROVED** [review]

File: src/main/java/helperInstrument/EnviromentHelper.java (hunk 3 above)
Comment (Oleg-Decik) [reply src/main/java/helperInstrument/EnviromentHelper.java:35]: actualy i don`t need it i will remove it

**Pull request closed without merging**